.circleci/config.yml

```diff
@@ -296,7 +296,27 @@ jobs:
           command: xcodebuild -scheme MapboxNavigation-Package test -destination "platform=iOS Simulator,OS=<< parameters.iOS >>,name=<< parameters.device >>" | xcpretty
 
 workflows:
-  workflow:
+  extended-workflow:
+    jobs:
+      - spm-test-job:
+          name: "swift test; Xcode 12.4; iOS 14.4"
+          xcode: "12.4.0"
+          iOS: "14.4"          
+          device: "iPhone 12 Pro Max"     
+      - spm-test-job:
+          name: "swift test; Xcode 12.4; iOS 13.7"
+          xcode: "12.4.0"
+          iOS: "13.7"          
+          device: "iPhone 11 Pro Max"
+    triggers:
+       - schedule:
+           cron: "0 0 * * *" # Once per day at 00:00
```

Comment:
Is there any reason to run CI build every day? I don't think we will monitor these builds every day. IMHO CI builds for every PR should be enough.

Reply:
Yes, the motivation is in the description. 

Visibility issue is a thing which can be solved with a slack bot integration.